src/Database/Schema/MysqlSchema.php

```diff
@@ -389,10 +389,14 @@ public function columnSql(TableSchema $schema, $name)
         }
 
         $hasPrecision = [TableSchema::TYPE_FLOAT, TableSchema::TYPE_DECIMAL];
-        if (in_array($data['type'], $hasPrecision, true) &&
-            (isset($data['length']) || isset($data['precision']))
-        ) {
-            $out .= '(' . (int)$data['length'] . ',' . (int)$data['precision'] . ')';
+        if (in_array($data['type'], $hasPrecision, true)) {
+            if (isset($data['length'])) {
```

Comment:
This check can be combined with check in outer `if` itself and avoid an nested `if`.